src/Components/Episode/Episode.css

```diff
@@ -53,6 +52,16 @@
     background-color: #01b7cc;
 }
 
+.episodePlayIcon {
```

Comment:
since these playIcons are used on multiple pages it might be best to bring all the styling for them to a central location